plutus-core/plutus-core/examples/PlutusCore/Examples/Builtins.hs

```diff
@@ -244,40 +257,39 @@ instance uni ~ DefaultUni => ToBuiltinMeaning uni ExtensionFun where
         commaPlc
             :: SomeConstant uni a
             -> SomeConstant uni b
-            -> SomeConstantPoly uni (,) '[a, b]
+            -> SomeConstant uni (a, b)
```

Comment:
Quite nicer, isn't it?

Reply:
The rest of the file does look quite a bit more readable.